styles/widgets/common/dropDownEditor.less

```diff
@@ -56,6 +56,12 @@
     }
 }
 
+.dx-placeholder:before {
```

Comment:
I think we should move it to the [textEditor](https://github.com/mrjohn0011/DevExtreme/blob/dde_placeholder/styles/widgets/common/textEditor.less#L28) common styles